talentmap_api/fsbid/views/publishable_positions.py

```
@@ -69,5 +69,11 @@ def get(self, request, *args, **kwargs):
 
         return Response(result)
 
+class FSBidPublishablePositionsCSVView(BaseView):
+    permission_classes = [Or(isDjangoGroupMember('ao_user'), isDjangoGroupMember('bureau_user')), isDjangoGroupMember('superuser'), ]
 
-
+    def get(self, request):
+        '''
+        Exports Publishable Positions to CSV format
+        '''
+        return services.get_publishable_positions_csv(request.query_params, request.META['HTTP_JWT'], f"{request.scheme}://{request.get_host()}")
```

so, with our other backoffice endpoint calls - we've been adding error handling in the views... 

right now - if this call fails, it just kinda fails silently ... i think you should add a toast to the Redux Action on the frontend to notify the user that the call failed (in case it does) 

I'm not sure if you NEED to add handling here, as well - it's helpful for bug fixing to add something more than the generic 500 error you get, but as of now - the page won't crash so 🤷 (can test this by changing the Proc Name to something that won't return)